src/nvim/regexp.c

```diff
@@ -6914,13 +6914,8 @@ static regengine_T bt_regengine =
   bt_regcomp,
   bt_regfree,
   bt_regexec_nl,
-  bt_regexec_multi
-#ifdef REGEXP_DEBUG
-  ,(char_u *)""
-#endif
-#ifdef DEBUG
```

Comment:
This `#ifdef DEBUG` was introduced in 3f29a02166878bdbf32b0d638aa0e0c9d83a73cc to silence a compiler warning. Now it's gone again.